librz/debug/p/debug_winkd.c

```diff
@@ -337,6 +382,56 @@ static RzList *rz_debug_winkd_modules(RzDebug *dbg) {
 	return ret;
 }
 
+#include "native/bt/windows-x64.c"
+
+RzList *rz_debug_winkd_frames(RzDebug *dbg, ut64 at) {
+	if (!kdctx || !kdctx->desc || !kdctx->syncd) {
+		return NULL;
+	}
+	RzList *ret = NULL;
+	if (kdctx->windctx.is_arm) {
+		// TODO
+	} else {
+		if (kdctx->windctx.is_64bit) {
+			struct context_type_amd64 context = { 0 };
+			backtrace_windows_x64(dbg, &ret, &context);
+		} else {
```

Comment:
It doesn't support x86 32 bytes?
I wonder also, if it makes sense to print warning that it's not implemented, also for ARM.

Reply:
Implemented in bff25a8c7263e375e17796fee2ea4fa84380a41c